Product/Production/Gateway/DocumentQuery_30/src/main/resources/docquery/_30/webservices.xml

```diff
@@ -140,7 +140,7 @@
     </jaxws:endpoint>
     <jaxws:endpoint xmlns:edq="urn:gov:hhs:fha:nhinc:dq:entitydeferredresponse" id="EntityDocQueryDeferredResult"
         address="/EntityService/EntityDeferredResultOptionResponse" serviceName="edq:EntityDocQueryDeferredResult" endpointName="edq:EntityDocQueryDeferredResultPortSoap"
-        implementor="gov.hhs.fha.nhinc.docquery.deferred.entity.EntityDeferredResultsOption"
+        implementor="gov.hhs.fha.nhinc.deferredresults.entity.EntityDeferredResultsOption"
```

Comment:
Should have "implementorClass" set to the interface to explicitly state the class type for injection